action/action_test.go

```diff
@@ -88,8 +88,14 @@ func TestAction(t *testing.T) {
 		t.Fatalf("Error: %s", err)
 	}
 
-	if as := act.String(); as != "Store(Path: "+filepath.Join(td, "store")+", Mounts: )" {
-		t.Errorf("act.String(): %s", as)
+	if as := act.Name; as != "action.test" {
+		t.Errorf("act.Name: %s", as)
+	}
+	if as := act.Store.Path(); strings.Index(as, filepath.Join(td, "store")) == -1 {
```

Comment:
using `strings.Index` is a small hax that makes it work on both mac and linux ;)

Reply:
I think it would be more idiomatic to use `!strings.Contains(as, ...` instead.